src/Services/BaseService.php

```diff
@@ -106,6 +107,10 @@ protected function buildUpdateColumns($passed_in = array())
             if (in_array($key, array_column($this->autoIncrements, 'Field'))) {
                 continue;
             }
+            // exclude uuid columns
```

Comment:
The uuid columns should be considered immutable and not available for updates.
If for some reason it could be updateable you would need to implement the uuid conversions for things to work correctly.